docs/jwst/cube_build/reference.rst

```diff
@@ -3,12 +3,12 @@ Reference File
 
 There are two types of reference files used by the cube_build step. The first type holds the default
 cube parameters used in setting up the output IFU Cube. The reftype for this reference file is *cubepars*
-and there is a  reference file of this type for MIRI data and one for NIRSPEC data. These files contain a table 
-for each band of the spatial and spectral 
-size to use to construct the IFU Cube. For build 7.1, if more than one band is used to build the IFU cube,
+and there is a  reference file of this type for MIRI data and one for NIRSPEC data. These files contain  tables 
+for each band of the spatial and spectral size and the size of the region of interest to use to construct the 
+IFU cube. For build 7.1, if more than one band is used to build the IFU cube,
```

Comment:
We're well past build 7.1 now. You could probably leave out that reference, i.e. just change the sentence to start with "If more than one band is used ..."